packages/block-editor/src/components/block-inspector/style.scss

```diff
@@ -1,4 +1,15 @@
 .block-editor-block-inspector {
+	p {
```

Comment:
It feels like adding so general styles to the `BlockInspector` might have unpredicted consequences on the UI outside of the Customizer.

Reply:
I remember seeing these styles before elsewhere in the codebase. They're styles that the Interface package would normally add:
https://github.com/WordPress/gutenberg/blob/trunk/packages/interface/src/components/complementary-area/style.scss#L34-L43

They probably shouldn't be there either.

Reply:
Yeah, these are the styles already applied to the inspector from `<ComplementaryArea>`. I was just trying to match the style here so that the inspector can have a consistent look when placing outside the complementary area.